lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -73,8 +83,17 @@ def create_client_properties
             username=#{username} \\
             password=#{password} ;
         CLIENT_PROPERTIES
+      end
 
-        File.write(client_properties_path, content)
+      def unsecure_client_properties_content(algorithm)
+        <<~CLIENT_PROPERTIES
+          ssl.endpoint.identification.algorithm=#{algorithm}
```

Comment:
Is this supposed to be here?

Reply:
Thank you for the feedback @bdunne  I appreciate it.
Yes `ssl.endpoint.identification.algorithm=#{algorithm}` is needed here to enable IPAddr resolution to work.